packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -42,8 +42,12 @@ import 'theme.dart';
 class BottomAppBar extends StatefulWidget {
   /// Creates a bottom application bar.
   ///
-  /// The [color], [elevation], and [clipBehavior] arguments must not be null.
+  /// The [clipBehavior] arguments must not be null.
   /// Additionally, [elevation] must be non-negative.
+  ///
+  /// If [color], [elevation], or [shape] arguments are null, they will be
```

Comment:
you can leave out "arguments" here, since you're only referring to arguments.

their [BottomAppBarTheme] values will be used. If the corresponding [BottomAppBar] property is null then the default specified in the property's documentation will be used.

Reply:
Done.